xpp3/xpp3_min/pom.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>xpp3</groupId>
+    <artifactId>xpp3-parent</artifactId>
+    <version>1.1.4-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>xpp3_min</artifactId>
+  <packaging>bundle</packaging><!-- bundle = jar + OSGi metadata -->
+
+  <name>MXP1: Xml Pull Parser 3rd Edition (XPP3) Minimal implementation</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>xmlpull</groupId>
+      <artifactId>xmlpull</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <scope>test</scope>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <configuration>
+          <instructions>
+            <Bundle-SymbolicName>org.xmlpull.mxp1</Bundle-SymbolicName>
```

Comment:
@ge0ffrey I am not 100% sure about this. It follows the package name, but it seems `xpp3_min` is a more widely known name. WDYT? Would `org.xmlpull.xpp3_min` be better?